consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/statemachine/QbftBlockHeightManagerFactory.java

```diff
@@ -26,19 +27,24 @@
   private final BftFinalState finalState;
   private final MessageValidatorFactory messageValidatorFactory;
   private final MessageFactory messageFactory;
+  private final ValidatorModeTransitionLogger qbftTransitionNotifier;
 
   public QbftBlockHeightManagerFactory(
       final BftFinalState finalState,
       final QbftRoundFactory roundFactory,
       final MessageValidatorFactory messageValidatorFactory,
-      final MessageFactory messageFactory) {
+      final MessageFactory messageFactory,
+      final ValidatorModeTransitionLogger validatorModeTransitionLogger) {
     this.roundFactory = roundFactory;
     this.finalState = finalState;
     this.messageValidatorFactory = messageValidatorFactory;
     this.messageFactory = messageFactory;
+    this.qbftTransitionNotifier = validatorModeTransitionLogger;
```

Comment:
can the field name be updated to match the constructor arg

Reply:
ops!